src/memory_region.rs

```diff
@@ -152,22 +155,19 @@ impl MemoryMapping {
         vm_addr: u64,
         len: u64,
     ) -> EbpfError<E> {
-        let mut regions_string = "".to_string();
-        if !self.regions.is_empty() {
-            regions_string = "regions:".to_string();
-            for region in self.regions.iter() {
-                regions_string = format!(
-                    "  {} \n{:#x} {:#x} {:#x}",
-                    regions_string, region.host_addr, region.vm_addr, region.len,
-                );
-            }
-        }
+        let region_name = match vm_addr & !(ebpf::MM_PROGRAM_START - 1) {
```

Comment:
This look up needs to be a little more sophisticated for stack violations since the stack could be violated on either side of the frame.  I've seen stack violations with address `0x1fffffff8`.  I was thinking in addition to all `0x200000000` violations we also include a frame size worth under `0x200000000`.

Reply:
If we assume that no section is filled to the top, then we can just offset the rounding threshold by a fixed amount for all sections. That way accesses to the high end of any section would be counted as belonging to the beginning of the next section.

Reply:
Not sure we want to do that for all sections.  I think the stack is probably the exception

Reply:
Tried to implement it but: We don't know `config.stack_frame_size` in `MemoryMapping`

Reply:
Yeah, that would have to be passed in, maybe during MemoryMapping's new...